scratch/benchmarks/remote.py

```diff
@@ -121,7 +121,8 @@ async def run_tasks(
     image
         Docker image to pull before running
     port_base
-        Port number for the first task. The tasks must use consecutive ports.
+        Port number for the first task. The tasks must use consecutive ports. If
+        None, the server is assumed to be immediately ready.
```

Comment:
Did you add/update this as part of your recent testing? Or was this always the case?